<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceOperator</class>
 <widget class="QWidget" name="DeviceOperator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_id">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_a" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="3">
           <widget class="QLabel" name="lbl_a_step">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="lbl_a_boost">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="chk_a_auto">
            <property name="text">
             <string>自动切换</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QPushButton" name="btn_a_increase">
            <property name="text">
             <string>升高</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLCDNumber" name="lcd_a_strength">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="digitCount">
             <number>3</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="3">
           <widget class="QPushButton" name="btn_a_boost">
            <property name="text">
             <string>按住执行</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSlider" name="sb_a_boost">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="lbl_cap_a_boost">
            <property name="text">
             <string>突增</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lbl_cap_a_step">
            <property name="text">
             <string>调节步进</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="lbl_cap_a">
            <property name="text">
             <string>A 通道</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="btn_a_decrease">
            <property name="text">
             <string>降低</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QComboBox" name="cb_a_wave"/>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QPushButton" name="btn_a_stop">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="4">
           <widget class="QSlider" name="silde_a_step">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>274</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="4">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="sp_a_auto_timer">
            <property name="suffix">
             <string>秒</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>600</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_b" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="0">
           <widget class="QLabel" name="lbl_cap_b_step">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;调节步进&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QComboBox" name="cb_b_wave"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_cap_b">
            <property name="text">
             <string>B 通道</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="btn_b_increase">
            <property name="text">
             <string>升高</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSlider" name="sb_b_boost">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="lbl_b_boost">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QPushButton" name="btn_b_boost">
            <property name="text">
             <string>按住执行</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QPushButton" name="btn_b_stop">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="chk_b_auto">
            <property name="text">
             <string>自动切换</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="lbl_b_step">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLCDNumber" name="lcd_b_strength">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="digitCount">
             <number>3</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lbl_cap_b_boost">
            <property name="text">
             <string>突增</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QSlider" name="silde_b_step">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>274</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="btn_b_decrease">
            <property name="text">
             <string>降低</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="4">
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="sp_b_auto_timer">
            <property name="suffix">
             <string>秒</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>600</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
